tests/cypress/integration/actions_tasks_objects/registration_involved/case_69_filters_sorting_jobs.js

```diff
@@ -0,0 +1,207 @@
+                        .find('[type="checkbox"]')
+                        .should('not.be.checked')
+                        .check()
+                        .should('be.checked');
+                    cy.contains('[type="button"]', 'OK').click();
+                } else {
+                    cy.contains('[type="button"]', 'Reset').click();
+                }
+            });
+    }
+
+    before(() => {
+        // Preparing a jobs
+        cy.visit('auth/register');
+        cy.userRegistration(
+            secondUser.firstName,
+            secondUser.lastName,
+            secondUserName,
+            secondUser.emailAddr,
+            secondUser.password,
+        );
+        cy.logout(secondUserName);
+        cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, labelName, imagesCount);
+        cy.createZipArchive(directoryToArchive, archivePath);
+        cy.login();
+        cy.createAnnotationTask(
+            taskName,
+            labelName,
+            attrName,
+            textDefaultValue,
+            archiveName,
+            false,
+            advancedConfigurationParams,
+        );
+        cy.openTask(taskName);
+        cy.assignJobToUser(0, secondUserName);
+        cy.assignJobToUser(1, secondUserName);
+        cy.logout();
+
+        // The first job is transferred to the validation status
+        cy.login(secondUserName, secondUser.password);
+        cy.openTaskJob(taskName);
+        cy.changeWorkspace('Review');
+        cy.createIssueFromControlButton(createIssueRectangle);
+        cy.interactMenu('Request a review');
+        cy.get('.cvat-request-review-dialog')
+            .should('exist')
+            .within(() => {
+                cy.get('.cvat-user-search-field')
+                    .find('[type="search"]')
+                    .type(`${Cypress.env('user')}{Enter}`);
+                cy.contains('[type="button"]', 'Submit').click();
+            });
+        cy.checkJobStatus(0, 'validation', secondUserName, Cypress.env('user'));
+
+        // The first job is transferred to the complete status
+        cy.openJob(1);
+        cy.changeWorkspace('Review');
+        cy.createIssueFromControlButton(createIssueRectangle);
+        cy.interactMenu('Request a review');
+        cy.get('.cvat-request-review-dialog')
+            .should('exist')
+            .within(() => {
+                cy.get('.cvat-user-search-field')
+                    .find('[type="search"]')
+                    .type(`${Cypress.env('user')}{Enter}`);
+                cy.contains('[type="button"]', 'Submit').click();
+            });
+        cy.logout(secondUserName);
+        cy.login();
+        cy.openTaskJob(taskName, 1);
+        cy.resolveIssue('.cvat-hidden-issue-label', 'Done');
+        cy.interactMenu('Submit the review');
+        cy.submitReview('Accept');
+        cy.checkJobStatus(1, 'completed', secondUserName, Cypress.env('user'));
```

Comment:
I am not sure I unrestand this code correctly. 

Why do we need to put the same job firstly to review stage, secondly to completed stage in before step?
Why do we need to create/resolve issues in this test? 
If we need a job in completed stage why do not press "Finish the job" (putting it to "completed") instead of temporary "Review" stage?

Reply:
Thanks. Reworked it.